CONTRIBUTING.md

```diff
@@ -93,6 +99,12 @@ To simplify iteration, you can also run in `watch` mode, which automatically re-
 
 To iterate quickly, simply re-build the project in IntelliJ after packaging is complete. Project resources will be hot-reloaded and changes are reflected on browser refresh.
 
+To iterate even quicker you can use `yarn serve` to spin up web server which will track source changes and compile code for web page in real time. Assuming you are working in the `webapp` folder:
+
+    cd git/presto/presto-main/src/main/resources/webapp/src
+    yarn serve --env=apiPort=8081
+`yarn` will show the http address to go to view the web UI (use it instead of the one from the presto server output): `http://localhost:8080/`.
```

Comment:
```suggestion
`yarn` shows the http address to view the web UI: `http://localhost:8080/`.
```
Nit: for better understanding, tell the user what they need and not about what they don't need. The "don't need" part can be a distraction.

Reply:
Agree with @steveburnett 
It would be good to explain the `--env=apiPort=<port number>` and `--end=apiHost=<IP address>` more here.  The example command `yarn serve --env=apiPort=8081` may not work if the Presto server is not listening on 8081 and localhost. Since you are editing this part, could you add something like this:

Uses the following parameters to specify the Presto Server's information:
- `---env=apiHost=<IP address>`: Specify the Presto Server IP Address. The default value is `127.0.0.1`
- `---end=apiPort=<Port Number>`: Specify the Presto Server port number. The default value is `8080`